zipkin-autoconfigure/collector-kafka10/pom.xml

```diff
@@ -43,22 +43,23 @@
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-maven-plugin</artifactId>
         <version>${spring-boot.version}</version>
-        <executions>
-          <execution>
-            <goals>
-              <goal>repackage</goal>
```

Comment:
is repackage redundant? or is there another reason why this isn't needed anymore?

Reply:
IMHO repackage is not needed. Is there a situation where we need to run this as a standalone jar? If we need to we have to provide the launcher in the layout factory as optional.

Reply:
nope.. just asking!